labs-ios-starter/Cities/CitiesViewController.swift

```diff
@@ -21,7 +21,9 @@ extension CitiesViewController: UICollectionViewDelegate, UICollectionViewDataSo
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "cityCell", for: indexPath) as! CityCollectionViewCell
         cell.cityStateLabel.text = "Chicago, IL"
         cell.populationLabel.text = "Population: 2.7M"
-        cell.cityImageView.image = UIImage(systemName: "building.2.fill")
+        cell.cityImageView.image = UIImage(named: "chicago")
+        cell.cityImageView.contentMode = .scaleToFill
```

Comment:
let's change this to .aspectfit so the image doesn't get distorted or even aspect fill if you want the whole image view to be used.